Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -1241,7 +1245,10 @@ void Renderer::SetViewport()
 		};
 		glViewport(iceilf(X), iceilf(Y), iceilf(Width), iceilf(Height));
 	}
-	glDepthRangef(GLNear, GLFar);
+	if (g_Config.backend_info.bSupportsClipControl)
+		glDepthRangef(GLFar, GLNear);
+	else
+		glDepthRangef(GLNear, GLFar);
```

Comment:
I think it's easier to check the shader formulation to also match "glDepthRangef(GLFar, GLNear);" without clip control